src/proposals/Proposals.jl

```diff
@@ -138,5 +139,97 @@ function (prop::RWalk)(rng::AbstractRNG,
     return u, v, logl, ncall
 end
 
+"""
+    Proposals.RStagger(;ratio=0.5, walks=25, scale=1)
+
+Propose a new live point by random staggering away from an existing live point. 
+This differs from the random walk proposal in that the step size here is exponentially adjusted
+to reach a target acceptance rate _during_ each proposal, in addition to _between_
+proposals.
+
+`ratio` is the target acceptance ratio, `walks` is the minimum number of steps to take, and `scale` is the proposal distribution scale, which will update _between_ proposals.
+"""
+
+@with_kw mutable struct RStagger <: AbstractProposal
+    ratio = 0.5
+    walks = 25
+    scale = 1.0
+end
 
+function (prop::RStagger)(rng::AbstractRNG,
+    point::AbstractVector,
+    logl_star,
+    bounds::AbstractBoundingSpace,
+    loglike,
+    prior_transform;
+    kwargs...)
```

Comment:
Style: Some extra whitespace here helps delineate the function args from the function body

```suggestion
        point::AbstractVector,
        logl_star,
        bounds::AbstractBoundingSpace,
        loglike,
        prior_transform;
        kwargs...)
```